pci.ids

```diff
@@ -9,6 +9,117 @@
 #	device  device_name				<-- single tab
 #		subvendor subdevice  subsystem_name	<-- two tabs
 
+# ETH Cards
+#---------
+
+1000  Broadcom / LSI
+	16d7  BCM57414 NetXtreme-E 10Gb/25Gb RDMA Ethernet Controller
+		14e4 1402  BCM957414A4142CC 10Gb/25Gb Ethernet PCIe
+14e4  Broadcom Inc. and subsidiaries
+	1751  BCM57504 NetXtreme-E 10Gb/25Gb/40Gb/50Gb/100Gb/200Gb Ethernet
+15b3  Mellanox Technologies
+	1015  MT27710 Family [ConnectX-4 Lx]
+		15b3 0003  Stand-up ConnectX-4 Lx EN, 25GbE dual-port SFP28, PCIe3.0 x8, MCX4121A-ACAT
+	101d  MT2892 Family [ConnectX-6 Dx]
+8086  Intel Corporation
+	1521  I350 Gigabit Network Connection
+	1533  I210 Gigabit Network Connection
+	1539  I211 Gigabit Network Connection
+	1572  Ethernet Controller X710 for 10GbE SFP+
+		1137 020a  Ethernet Converged NIC X710-DA2
+		1137 013b  Ethernet Converged NIC X710-DA4
+		1590 0000  Ethernet Controller X710 for 10GbE SFP+
+	1593  Ethernet Controller E810-C for SFP
+		1137 02be  E810XXVDA2 2x25/10 GbE SFP28 PCIe NIC
+	37d0  Ethernet Connection X722 for 10GbE SFP+
+
+# FC adapters
+#-----------
+
+1000  Broadcom / LSI
+10df  Emulex Corporation
+	e131  LightPulse 8Gb/s PCIe Shared I/O Fibre Channel Adapter
+	e300  LPe31000/LPe32000 Series 16Gb/32Gb Fibre Channel Adapter
+		10df e300  LPe32002-M2 2-Port 32Gb Fibre Channel Adapter
+1b4b  Marvell Technology Group Ltd.
+1fc1  QLogic, Corp.
+	2071  ISP2714-based 16/32Gb Fibre Channel to PCIe Adapter
+		1077 029e  QLE2694 Quad Port 16Gb Fibre Channel to PCIe Adapter
+		1077 0283  QLE2764 Quad Port 32Gb Fibre Channel to PCIe Adapter
+	2261  ISP2722-based 16/32Gb Fibre Channel to PCIe Adapter
+		1077 02a7  QLE2690 Single Port 16Gb FC to PCIe Gen3 x8 Adapter
+		1077 02a8  QLE2692 Dual Port 16Gb FC to PCIe Gen3 x8 Adapter
+		1077 0299  QLE2740 Single Port 32Gb Fibre Channel to PCIe Adapter
+		1077 029a  QLE2742 Dual Port 32Gb Fibre Channel to PCIe Adapter
+	2532  ISP2532-based 8Gb Fibre Channel to PCI Express HBA
+		1077 015d  QLE2562 PCI Express to 8Gb FC Dual Channel
+		1077 015e  QLE2564 PCI Express to 8Gb FC Quad Channel
+
+# Accelerators
+#------------
+
+10de  NVIDIA Corporation
+	1db6  GV100GL [Tesla V100 PCIe 32GB]
+	1df6  GV100GL [Tesla V100S PCIe 32GB]
+	1eb8  TU104GL [Tesla T4]
+
```

Comment:
There is ATI device found on Richelieu 0x1002:0x683f (Radeon HD 7750/8740 / R7 250E)

Reply:
The fact that it is installed in a development rig doesn't automatically mean that it's validated by QA.